website/js/index.js

```diff
@@ -0,0 +1,21 @@
+$(".imgAdd").click(function(){
```

Comment:
You can should doing this without jQuery - it'll likely be much simpler code: https://developers.google.com/web/fundamentals/media/capturing-images/

Reply:
e.g.

```js
<input type="file" accept="image/*" id="file-input">
<script>
  const fileInput = document.getElementById('file-input');

  fileInput.addEventListener('change', (e) => doSomethingWithFiles(e.target.files));
</script>
```